tasks/build.js

```diff
@@ -84,7 +84,9 @@ module.exports = function(options) {
 	});
 
 	gulp.task('build:compile-lib-sass', function(cb) {
+		var gulpIf = themeUtil.requireDependency('gulp-if', themeConfig.version);
```

Comment:
The only need for the `requireDependency` method is for dependencies that are loaded via a preset (https://www.npmjs.com/package/liferay-theme-deps-7.0).